File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_1/The solution/Java/Q_1.java
package com.company;

import java.util.Scanner;

public class Q_1 {

    public static void main(String[] args) {
        int num;
        Scanner input = new Scanner(System.in);
        
        // Read number from user
        System.out.print("Please enter number Of type integer: ");
        num = input.nextInt();

        // Check number positive or negative
        if(num >= 0)
            System.out.println("Number is positive");
        else
            System.out.println("Number is negative");

        // Check number even or odd
        if(num % 2 == 0)
            System.out.println("And number is even");
        else
            System.out.println("And number is odd");
    }
}

File: Level_1/Term_2/Progrmming 2/Ramadan-Questions/Day_1/The solution/CPP/Q_1.cpp
#include <iostream>
using namespace std;

int main()
{
    int num;

    // Read number from user
    cout << "Please enter number Of type integer: ";
    cin >> num;

    // Check number positive or negative
    if(num > 0)
        cout << "Number is positive" << endl;
    else
        cout << "Number is negative" << endl;

    // Check number even or odd
    if(num % 2 == 0)
        cout << "And number is even" << endl;
    else
        cout << "And number is odd" << endl;
    return 0;
}
